src/app/admin/filters.js

```diff
@@ -0,0 +1,61 @@
+import { SearchInput, DateTimeInput, SelectInput } from 'react-admin';
+import { QuickFilter } from './_components';
+
+export const specialistsFilters = [
```

Comment:
🟠 aren't those filters suppose to be in `app/admin/components/{resource}/` folder?